exercises/practice/mazy-mice/.docs/instructions.md

```diff
@@ -0,0 +1,53 @@
+## Introduction
+
+Meet Mickey and Minerva, two clever mice who love to navigate their way through a maze to find cheese. 
+They enjoy a good challenge, but with only their tiny mouse brains, they prefer if there's only one correct path to the cheese.
+
+## Instructions
+
+Your task is to generate the perfect mazes for Mickey and Minerva — those with only one solution and no isolated sections. 
+Here's what you need to know:
+
+- The maze is rectangular, with the outer borders roughly forming a rectangle.
+- The maze has square cells and orthogonal passages intersecting at right angles.
+- The program should accept two parameters: rows and columns. The maze should be between 5 and 100 cells in size.
+- Consider the maze walls by using the formula 2x+1 for the actual width and height, where x is rows or columns.
+- If no seed is provided, generate a random maze. If a seed is provided, get the same labyrinth every time.
+- Use [box-drawing][Box-drawing] characters to draw walls, and an arrow symbol (⇨) for the entrance on the left and exit on the right.
```

Comment:
I'm not entirely sure what would be expected of me after reading this.
I think it would help to be more explicit about what types of characters are allowed and such.

Reply:
We can add a similar table to the task description:

| Character | Name                                  | Unicode |
|:---------:|:--------------------------------------|:--------|
|     ┌     | box drawings light down and right     |  U+250C |
|     ─     | box drawings light horizontal         |  U+2500 |
|     ┬     | box drawings light down and horizontal|  U+252C |
|     ┐     | box drawings light down and left      |  U+2510 |
|     │     | box drawings light vertical           |  U+2502 |
|     └     | box drawings light up and right       |  U+2514 |
|     ┴     | box drawings light up and horizontal  |  U+2534 |
|     ┘     | box drawings light up and left        |  U+2518 |
|     ├     | box drawings light vertical and right |  U+2520 |
|     ⇨     | rightwards white arrow                |  U+21E8 |

Reply:
I would like to point out that there are two examples in the task description where all graphic symbols are included. As an alternative solution, we can add a similar table to the hint file.

Reply:
I've added the table to the hints.